Common.md

```diff
@@ -20,3 +42,140 @@ Language names are title cased.
+The collective application of a set of instruments to comprehensively monitor a computing system is the instrumentation.
+
+For example:
+
+- Instrumentation for an HTTP library might be the dedicated code in the library that measure its performance (e.g. latency, throughput, errors).
+- Instrumentation for a cluster of Kubernetes pods might be the health check probes.
+- Instrumentation for a computer's CPU might be the embedded sensors measuring its physical state (e.g temperature, speed, power).
+
+Including attributes to identify what instrumentation is used is crucial for interoperability between telemetry systems and the New Relic platform.
+
+#### `instrumentation.provider`
+
+All exported measurements MUST contain an `instrumentation.provider` attribute that identifies the provider of instrumentation.
+
+Example values of this attribute are:
+
+* `newRelic`
+* `prometheus`
+* `dropwizard`
+* `opencensus`
+* `opentelemetry`
+
+#### `instrumentation.name`
+
+All exported measurements SHOULD contain an `instrumentation.name` attribute that identifies the code that is being instrumented if the instrumentation provider exposes this information, otherwise it SHOULD be omitted.
+This would be the framework, library, or application that uses the instrumentation to report measurements.
+
+Example values of this attribute are:
+
+* `spring-framework`
+* `python-twisted`
+* `gorilla/mux`
+
+#### `instrumentation.version`
+
+All exported measurements SHOULD contain an `instrumentation.version` attribute set to the version of the instrumentation if the instrumentation provider expose this information, otherwise it SHOULD be omitted.
+
+Example values of this attribute are:
+
+* `v0.1.0`
+* `3`
+* `alpha-v1`
+
+### Collector
+
+Identifying what collected and transmitted the measurements of a system is important to know.
+It allows the New Relic platform to inform customers what telemetry systems were used to transmit their data, something that can help debug issues and provide insight into data flows.
+
+A collector can be as simple as the exporter itself, or as complex as a hierarchical system of proxies.
+Regardless of the complexity or simplicity of the collectors, the measurements they transmit need to identify what collector was the ultimate transmitter.
+The following attributes are used to make this identification possible.
+
+#### `collector.name`
+
+If the exporter is transmitting to New Relic directly than all exported measurements SHOULD contain a `collector.name` attribute set to the name of the exporter, otherwise it SHOULD be omitted.
+
+Example values of this attribute are:
+
+* `newrelic-dropwizard-reporter`
+* `newrelic-opentelemetry-go-exporter`
+
+#### `collector.version`
+
+If the exporter is transmitting to New Relic directly than all exported measurements SHOULD contain a `collector.version` attribute set to the version of the exporter, otherwise it SHOULD be omitted.
```

Comment:
```suggestion
If the exporter is transmitting to New Relic directly then all exported telemetry SHOULD contain a `collector.version` attribute set to the version of the exporter, otherwise it SHOULD be omitted.
```

Reply:
+1 to changing to `then`, but I'm not sure about the `s/measurements/telemetry`. Measurements are the things containing attributes not telemetry. Telemetry is the process of collecting and transmitting measurements or other data to receiving endpoints.

Reply:
Ah, I tend to use the 2nd definition here: https://www.merriam-webster.com/dictionary/telemetry

But, I'm fine with keeping measurements.